Source/automap.cpp

```diff
@@ -351,6 +351,7 @@ void DrawAutomapPlr(CelOutputBuffer out, int pnum)
 		DrawLineTo(out, x + AmLine16, y + AmLine8, x + AmLine8, y + AmLine8, playerColor);
 		break;
 	case DIR_S:
+	case DIR_OMNI:

```

Comment:
This looks wrong to me. Shouldn't we just throw some sort of error in this case? Unless the value is the same as `DIR_S`?

Reply:
It would be incorrect for the player to have DIR_OMNI as that is the unitialized direction meaningn they aren't looking in any direction (not possible for real people). But since it's a rather minor issues I think it's to agressive to terminate the application, so I was thinking it would be ok to just assume that they are looking in the direction that they spawn in.

Reply:
Thanks for clarifying. We'll make these constraints stronger over time so that invalid values are made impossible.